contracts/container/contract.go

```diff
@@ -1105,38 +1105,38 @@ func PutReport(cid interop.Hash256, sizeBytes, objsNumber int, pubKey interop.Pu
 			storage.Put(ctx, append(reportersKey, counterToBytes(index)...), std.Serialize(reporter))
 		}
 	} else {
-		estInfo = ContainerEstimation{
-			SizeEstimation:          sizeBytes,
-			ObjectsNumberEstimation: objsNumber,
+		cnrInfo = ContainerInfo{
+			ContainerSize:   sizeBytes,
+			NumberOfObjects: objsNumber,
 		}
 		reporter := NodeReport{PublicKey: pubKey, NumberOfReports: 1, ContainerSize: sizeBytes, NumberOfObjects: objsNumber}
 		storage.Put(ctx, append(reportersKey, counterToBytes(0)...), std.Serialize(reporter))
 	}
 
-	storage.Put(ctx, infoKey, std.Serialize(estInfo))
-	runtime.Log("saved container size estimation")
+	storage.Put(ctx, infoKey, std.Serialize(cnrInfo))
+	runtime.Log("saved container size report")
 }
 
-// GetEstimation method returns an average load container's storage
-// nodes have and reported with [PutReport].
+// GetContainerInfo method returns a summary container state based on all reports storage nodes
+// provided via [PutReport].
 //
 // If the container doesn't exist, it panics with [cst.NotFoundError]. If no
 // reports were claimed, it returns zero values.
-func GetEstimation(epoch int, cid interop.Hash256) ContainerEstimation {
+func GetContainerInfo(epoch int, cid interop.Hash256) ContainerInfo {
```

Comment:
`GetContainerDataInfo`/`GetContainerLoadInfo`/`GetContainerUseInfo`? `ContainerInfo` can be anything.